package.json

```diff
@@ -13,5 +13,11 @@
   "license": "MPL-2.0",
   "bugs": {
     "url": "https://github.com/mozilla/fxa-content-server-l10n/issues"
+  },
+  "devDependencies": {
+    "grunt": "0.4.5",
+    "grunt-l10n-lint": "0.0.0",
```

Comment:
I'm surprised and impressed that npm lets you register this as a package version....